insights/specs/default.py

```diff
@@ -686,8 +686,6 @@ class DefaultSpecs(Specs):
     x86_pti_enabled = simple_file("sys/kernel/debug/x86/pti_enabled")
     x86_retp_enabled = simple_file("sys/kernel/debug/x86/retp_enabled")
     # Suspend the collection for xfsprogs bug INSGHTCORE-257
```

Comment:
I think this comment should also be removed

Reply:
Good catch!

Sorry for missing it. Updated.